R/theme_Morons.R

```diff
@@ -0,0 +1,52 @@
+#' @title Morons Theme
+#'
+#' @description Define function for consistent plot theme.
+#'
+#' @description This function was set up to have a standardized ggplot theme for creating consistent-styled plots.
+#' The parameters in this function lets the user control theme parameters such as; font size, font style and legend positions.
+# For more info, refer to the parameter descriptions.
+#'
+#' @param base_size Size of the font on the plot. Defaults to 14.
+#' @param base_family Font family to be used on the plot. Defaults to Arial. Always use cairo device when saving the resulting plot!
+#' @param my_legend_position Where to draw the legend? Defaults to the bottom of the plot.
+#' @param my_legend_direction Which direction to draw the legend? Defaults to horizontal.
+#'
+#' @return Nothing.
+#'
+#' @import ggplot2
+#' @importFrom ggthemes theme_foundation
+#' @export
+#'
+#' @examples
+#' library(ggplot2)
+#' ggplot(mpg, aes(displ, hwy, colour = class)) +
+#' geom_point() +
+#' theme_Morons()
```

Comment:
This returns this error:
```
Error in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y,  : 
  invalid font type
In addition: There were 50 or more warnings (use warnings() to see the first 50)

> warnings()
Warning messages:
1: In grid.Call(C_stringMetric, as.graphicsAnnot(x$label)) :
  font family 'Arial' not found in PostScript font database
2: In grid.Call(C_stringMetric, as.graphicsAnnot(x$label)) :
  font family 'Arial' not found in PostScript font database
3: In grid.Call(C_stringMetric, as.graphicsAnnot(x$label)) :
  font family 'Arial' not found in PostScript font database
...
```

Reply:
This is likely related to Arial not being installed on your computer. Since the fonts are different between different OS, and Arial is not installed by default on Linux, this can be modified with the base_family argument of this function. Can you try either installing Arial, or using one of the fonts available on your machine as value for base_family?